autologout.module

```diff
@@ -189,34 +171,30 @@ function autologout_page_attachments(array &$page) {
 
   // Check if JS should be included on this request.
   if ($autologout_manager->autologoutPreventJs()) {
-
     return;
   }
 
-  // Check if anything wants to be refresh only. This URL would
-  // include the javascript but will keep the login alive whilst
-  // that page is opened.
+  // Check if anything wants to be refresh only. This URL would include the
+  // javascript but will keep the login alive whilst that page is opened.
   $refresh_only = $autologout_manager->autologoutRefreshOnly();
-
   $now = REQUEST_TIME;
+  $settings = \Drupal::config('autologout.settings');
 
   $timeout = $autologout_manager->autologoutGetUserTimeout();
-
-  $timeout_padding = \Drupal::config('autologout.settings')->get('padding');
-  $redirect_url = \Drupal::config('autologout.settings')->get('redirect_url');
+  $timeout_padding = $settings->get('padding');
+  $redirect_url = $settings->get('redirect_url');
   $redirect_query = \Drupal::service('redirect.destination')->getAsArray() + array('timeout' => 1);
-  $no_dialog = \Drupal::config('autologout.settings')->get('no_dialog');
-  $use_alt_logout_method = \Drupal::config('autologout.settings')->get('use_alt_logout_method');
-
+  $no_dialog = $settings->get('no_dialog');
+  $use_alt_logout_method = $settings->get('use_alt_logout_method');
 
   // Get all settings JS will need for dialog.
-  $msg = t('@msg', array('@msg' => \Drupal::config('autologout.settings')->get('message')));
+  $msg = t('@msg', array('@msg' => $settings->get('message')));
   $settings = array(
     'timeout' => $refresh_only ? ($timeout * 500) : ($timeout * 1000),
     'timeout_padding' => $timeout_padding * 1000,
     'message' => t('@msg', array('@msg' => $msg)),
     'redirect_url' => Url::fromUserInput($redirect_url, array('query' => $redirect_query))->toString(),
-    'title' => t('@name Alert', array('@name' => \Drupal::config('autologout.settings')->get('site_name'))),
+    'title' => t('@name Alert', array('@name' => $settings->get('site_name'))),
```

Comment:
While fixing I just replaced the `\Drupal::config('autologout.settings')` with `$settings`. Now while reviewing it seems this should be broken as AFAIK `site_name` is not the part of `autologout.settings`.